templates/support/index.html

```diff
@@ -2,84 +2,116 @@
 
 {% block title %}Enterprise open source support{% endblock %}
 
-{% block meta_description %}Ubuntu Pro offers a single, per-node packaging of the most comprehensive Linux and open source software, application, security and IaaS support in the industry, with OpenStack support and Kubernetes support included, as well as for open source applications like Cassandra, Elasticsearch, Postgres and more.{% endblock %}
+{% block meta_description %}Ubuntu Pro with Enterprise Open Source Support offers the most comprehensive Linux, open source infrastructure & applications security and support in the industry. Available at simple per-node, pricing, it supports infrastructure solutions such as OpenStack and Kubernetes, as well as applications such as Spark, MongoDB, Kafka&reg;, Postgres, MySQL and more.{% endblock %}
 
 {% block meta_copydoc %}https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#{% endblock meta_copydoc %}
 
+{% block body_class %}is-paper{% endblock body_class %}
+
 {% block content %}
-<section class="p-strip--suru is-dark">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h1>Ubuntu Pro</h1>
-      <h2 class="p-heading--4">Security, support and managed open source for enterprises</h2>
-      <p>Future-proof the full stack, from the data centre to containers to the database, <abbr title="Logging, Monitoring and Aggregation services">LMA</abbr>, server and cloud-native applications with open source software support from Canonical.</p>
-      <p>Ubuntu Pro is a single, per-node package of the most comprehensive enterprise security and support for open source infrastructure and applications, with managed service offerings available.</p>
+<section class="p-strip is-shallow u-no-margin--bottom">
+  <div class="row--25-75">
+    <div class="col">
+      <div class="p-section--shallow">
+        <h1 class="u-no-margin--bottom">Enterprise support <br /> for your full open source stack</h1>
+      </div>
+      <div class="p-section--shallow">
```

Comment:
please add `u-n-margin--bottom` here, otherwise the spacing between the divider and the buttons is too big